chart/templates/dag-processor/dag-processor-deployment.yaml

```
@@ -216,6 +230,41 @@ spec:
           {{- if .Values.dagProcessor.logGroomerSidecar.env }}
               {{- tpl (toYaml .Values.dagProcessor.logGroomerSidecar.env) $ | nindent 12 }}
           {{- end }}
+          
+          {{- if hasKey .Values.dagProcessor.logGroomerSidecar "livenessProbe" }}
```

Can't this be the following?
```suggestion
          {{- if .Values.dagProcessor.logGroomerSidecar.livenessProbe }}
```

---

Done